pkg/kademlia/kademlia.go

```diff
@@ -177,8 +175,8 @@ func (k *Kad) manage() {
 							k.logger.Debugf("could not remove peer from addressbook: %s", peer.String())
 						}
 					}
-					k.logger.Debugf("could not connect to peer from kademlia %s: %v", bzzAddr.String(), err)
-					k.logger.Warningf("could not connect to peer")
+					k.logger.Debugf("peer not reachable from kademlia not reachable %s: %v", bzzAddr.String(), err)
```

Comment:
this message says: `peer not reachable from kademlia not reachable: ....`. maybe `peer not reachable from kademlia %s: %v"` is enough